sdk/core/azure-core-amqp/src/impl/uamqp/amqp/session.cpp

```diff
@@ -151,7 +151,7 @@ namespace Azure { namespace Core { namespace Amqp { namespace _detail {
     return max;
   }
 
-  void SessionImpl::Begin()
+  void SessionImpl::Begin(Azure::Core::Context const&)
```

Comment:
There's context parameter, but we're not using it. Is this a mistake? If yes, please check for other similar instances (search for "Context const&)" and "Context const&,"?)

Reply:
It's not a mistake. See work item #6040.

Eventually it will be used, but right now the infrastructure to support it is missing.